_pages/dev/extending-checkout.md

```diff
@@ -394,6 +394,51 @@ provideConfig({
 
 **Note:** You can use this same approach to combine all the steps into a single-step checkout.
 
+## Multiple Checkout Flows
+
```

Comment:
Does the OPF library need to be installed to enable multiple checkout flows?

Reply:
No, multiple checkout flows strategy can be enabled even if OPF library has not been installed.